Import/project.mjs

```diff
@@ -0,0 +1,53 @@
+import express from "express"
+import cors from "cors"
+import {Project} from "./index.mjs"
+
+const router = express.Router()
+router.use(
+  cors({
+    methods: "GET",
+    allowedHeaders: [
+      "Content-Type",
+      "Content-Length",
+      "Allow",
+      "Authorization",
+      "Location",
+      "ETag",
+      "Connection",
+      "Keep-Alive",
+      "Date",
+      "Cache-Control",
+      "Last-Modified",
+      "Link",
+      "X-HTTP-Method-Override"
+    ],
+    exposedHeaders: "*",
+    origin: "*",
+    maxAge: "600"
+  })
+)
+
+router.get("/", async (req, res) => {
```

Comment:
A note this this is likely actually a part of the project route, like `/project/import?data=uri` or `/project/create?data=uri`